source/crud/tests/unified/db-aggregate.yml

```diff
@@ -0,0 +1,73 @@
+# This file was created automatically using mongodb-spec-converter.
+# Please review the generated file, then remove this notice.
+
+description: db-aggregate
+schemaVersion: '1.1'
+createEntities:
+  -
+    client:
+      id: &client0 client0
+      useMultipleMongoses: false
+      uriOptions:
+        retryReads: false
+      observeEvents:
+        - commandStartedEvent
+  -
+    database:
+      id: &database0 database0
+      client: client0
+      databaseName: &database_name admin
+  -
+    collection:
+      id: &collection0 collection0
+      database: database0
+      collectionName: &collection_name crud-v2
+runOnRequirements:
```

Comment:
This is relevant to @patrickfreed since he was in the process of adding `serverless: false` to the legacy CRUD test format. If we migrate these tests to the unified runner that should no longer be necessary; however, it'd require bumping the `schemaVersion` of some of these tests to incorporate his unified format changes.